.coafile

```diff
@@ -0,0 +1,41 @@
+[Default]
+ignore = .tox/**, .gitignore, .gitreview, .gitmodules, **/provisioning/roles/**
```

Comment:
Working on it. Sorry, shouldn't have blindly copied the coafile. One thing, is it fine to keep .tox/** and the check on .sh files?